### PR TITLE
Adapt scalability range spec to the CI results

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
Adapt scalability  test range spec to the observed CI results.

#### Which issue(s) this PR fixes:

Relates to #1912

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: test/scalability/default_rangespec.yaml [hunk 1]
```diff
@@ -1,15 +1,32 @@
...
+# - #1782768037514973184
+# - #1782772615836864512
+# - #1782775995984515072
 cmd:
-  maxWallMs: 3600_000 #1h 
-  maxUserMs: 3600_000
-  maxSysMs: 3600_000
-  maxrss: 1024_000 #1000MiB
+  # Average value 351116.4 (+/- 0.9%), setting at +5%
+  maxWallMs: 368672
```
Comment (alculquicondor) [inline test/scalability/default_rangespec.yaml:9]: Nit:
I would round up to the next second or even next minute.

File: test/scalability/default_rangespec.yaml [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+
+  # Average value 111500 (+/- 14%), setting at +20%
+  maxUserMs: 133800
+
+  # Average value 27875 (+/- 16%), setting at +20%
+  maxSysMs: 33450
+  
+  # Average value 445012 (+/- 0.3%), setting at +5%
+  maxrss: 467263
```
Comment (alculquicondor) [inline test/scalability/default_rangespec.yaml:18]:
```suggestion
  maxRSSKb: 467263
```
IIUC. Also, worth rounding up to the next Mb.

File: test/scalability/default_rangespec.yaml (hunk 1 above)
Comment (trasc) [reply test/scalability/default_rangespec.yaml]: done

File: test/scalability/default_rangespec.yaml (hunk 2 above)
Comment (trasc) [reply test/scalability/default_rangespec.yaml]: done

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**